app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/theme/Type.kt

```diff
@@ -0,0 +1,36 @@
+package hr.dice.filipbionda.tmdbpractice.ui.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+// Set of Material typography styles to start with
+val Typography =
+    Typography(
+        bodyLarge =
+            TextStyle(
+                fontFamily = FontFamily.Default,
+                fontWeight = FontWeight.Normal,
+                fontSize = 16.sp,
+                lineHeight = 24.sp,
+                letterSpacing = 0.5.sp,
+            ),
+    /* Other default text styles to override
+    titleLarge = TextStyle(
```

Comment:
Also, remove these comments they are unnecessary. Provide the values that you find in Figma. Feel free to add them to the best typography value you see fit and we will discuss it then

Reply:
done